### PR TITLE
Fixed controller can't handle more than 2 GET methods.

### PR DESCRIPTION
**Pull request by Jark**
Fixes issue #57.

### CODE REVIEW

File: src/WebServer.UnitTests/Rest/RestControllerRequestHandlerWithMultipleSimilarlyNamedMethods.cs [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        }
+
+        [TestMethod]
+        [DataRow("/Get", "/Get")]
+        [DataRow("/Get/tom", "/Get/{name}")]
+        [DataRow("/Get/henk/age/15", "/Get/{name}/age/{age}")]
+        public void GetRestMethods_WithControllerThatHasMethodsSortedByNumberOfParametersDescending_CanHandleRequest(string uri, string route)
+        {
+            var restHandler = new RestControllerRequestHandler();
+            restHandler.RegisterController<MethodsSortedByParametersAscendingController>();
```
Comment (tomkuijsten) [inline src/WebServer.UnitTests/Rest/RestControllerRequestHandlerWithMultipleSimilarlyNamedMethods.cs]: Should be the descending controller I gues? :)

**Comment by tomkuijsten** [comment]
I see some problems with the [FromContent] attribute, that could let the lookup fail as the number of parameters can be equal (although the uri differs).

File: src/WebServer.UnitTests/Rest/RestControllerRequestHandlerWithMultipleSimilarlyNamedMethods.cs (hunk 1 above)
Comment (Jark) [inline src/WebServer.UnitTests/Rest/RestControllerRequestHandlerWithMultipleSimilarlyNamedMethods.cs]: Yup! Thanks, this is exactly why pull requests exist!

**Comment by Jark** [comment]
@tomkuijsten with FromContent, I'm thinking of solving this via inheritance of RestMethodInfo so that the following method:

```
public class FromContentData 
{
    public string Name { get; set; }
    public int Age { get; set; }
}

public IGetResponse([FromContent] FromContentData fromContentData) {}
```

gets expanded to (internally in code):
```
public IGetResponse(string name, int age) {}
```

I think that would be the nicest way to solve this.

**Comment by Jark** [comment]
Hey @tomkuijsten, I was thinking about how to solve this and then I was wondering in what use-case the [FromContent] would cause an issue. 

Can you think of any use-cases?

Oh and I fixed the second test, it uses the descending controller now :)

**Comment by tomkuijsten** [comment]
I did create a unit-test which failed, but to be honest, that was kind off a weird test. I think it's safe to merge this PR and try to create a test with a valid scenario that fails.

**Pull request merged**